### PR TITLE
Creating RCs sequentially not in parallel in Load e2e test

### PR DESCRIPTION
**Pull request by piosz**
Will update with e2e result.

### CODE REVIEW

File: test/e2e/load.go [hunk 1]
```diff
@@ -147,8 +168,41 @@ func playWithRC(c *client.Client, wg *sync.WaitGroup, ns, name string, size int)
 }
 
 func runRCLoad(c *client.Client, wg *sync.WaitGroup, ns, groupName string, size, count int) {
-	By(fmt.Sprintf("Running %v Replication Controllers with size %v and playing with them", count, size))
 	for i := 1; i <= count; i++ {
 		go playWithRC(c, wg, ns, groupName+"-"+strconv.Itoa(i), size)
 	}
 }
+
+func createRCGroup(c *client.Client, ns, groupName string, size, count, rate int) {
```
Comment (fgrzadkowski) [inline test/e2e/load.go]: s/rate/batchSize

I think that rate usually is something like "something per second". This parameter controls how many RC you create in one batch.

Comment (fgrzadkowski) [inline test/e2e/load.go]: Please add a godoc

File: test/e2e/load.go [hunk 2]
```diff
@@ -147,8 +168,41 @@ func playWithRC(c *client.Client, wg *sync.WaitGroup, ns, name string, size int)
...
+				defer wg.Done()
+				name := groupName + "-" + strconv.Itoa(i)
+				expectNoError(RunRC(c, name, ns, image, size), fmt.Sprintf("creating rc %s in namespace %s for the first time", name, ns))
+			}(i)
+		}
+		wg.Wait()
+	}
+}
+
+func cleanRCGroup(c *client.Client, ns, groupName string, size, count int) {
```
Comment (fgrzadkowski) [inline test/e2e/load.go]: Please add a godoc

File: test/e2e/load.go [hunk 3]
```diff
@@ -33,11 +33,17 @@ import (
...
+	simulationTime    = 10 * time.Minute
+	smallRCSize       = 5
+	mediumRCSize      = 30
+	bigRCSize         = 250
+	smallRCGroupName  = "load-test-small-rc"
+	mediumRCGroupName = "load-test-medium-rc"
+	bigRCGroupName    = "load-test-big-rc"
+	smallRCRate       = 20
+	mediumRCRate      = 5
+	bigRCRate         = 1
```
Comment (fgrzadkowski) [inline test/e2e/load.go]: s/Rate/BatchSize

**Comment by fgrzadkowski** [comment]
LGTM. Only two minor comments.

File: test/e2e/load.go (hunk 3 above)
Comment (piosz) [inline test/e2e/load.go]: Done

File: test/e2e/load.go (hunk 1 above)
Comment (piosz) [inline test/e2e/load.go]: Done

File: test/e2e/load.go (hunk 2 above)
Comment (piosz) [inline test/e2e/load.go]: Done

**Pull request merged**